tools/dependency/BUILD

```diff
@@ -41,3 +41,25 @@ py_binary(
         requirement("envoy.dependency.pip_check"),
     ],
 )
+
+py_binary(
+    name = "validate",
+    srcs = ["validate.py"],
+    args = [
+        "$(location //bazel:all_repository_locations)",
+        "$(location //source/extensions:extensions_build_config)",
+    ],
+    data = [
+        "//bazel:all_repository_locations",
+        "//source/extensions:extensions_build_config",
+    ],
+    deps = [
+        "//tools/base:bazel_query",
+    ],
+)
+
+py_binary(
```

Comment:
There isn't `py_test`?

Reply:
this has the existing `validate_test` script set up - below and in ci (ci/do_ci.sh) so its testing the same as now i think